### PR TITLE
Position test helper

### PR DESCRIPTION
**Pull request by sir-dunxalot**
This PR adds a `assertTooltipPosition()` public test helper:

```js
assertTooltipPosition(assert, { side: 'top' });
```

```js
assertTooltipPosition(assert, {
  side: 'top',
  selector: '.test-tooltip',
});
```

```js
assertTooltipPosition(assert, {
  side: 'top',
  selector: '.test-tooltip',
  targetSelector: '.test-tooltip-target',
});
```

The addon's test suite now uses this test helper instead of the previous, private `assertPosition()` method.

### CODE REVIEW

**Comment by sir-dunxalot** [comment]
@a15n Please review and make sure this method of adding a public test helper is in accordance with your work in #114.

File: test-support/helpers/ember-tooltips.js [hunk 1]
```diff
@@ -87,3 +101,41 @@ export function assertTooltipVisible(assert, options={}) {
...
+  );
+
+  /* We make sure the side being tested is valid. We
+  use Ember.assert because assert is passed in from QUnit */
+
+  if (!sideIsValid) {
+    Ember.assert(`You must pass side like assertTooltipSide(assert, { side: 'top' }); Valid options for side are top, right, bottom, and left.`);
+  }
+
+  const expectedSide = options.side || 'top';
```
Comment (a15n) [inline test-support/helpers/ember-tooltips.js]: we already have `side` from line 106 I think. `options.side` could be replaced with `side`

File: test-support/helpers/ember-tooltips.js [hunk 2]
```diff
@@ -87,3 +101,41 @@ export function assertTooltipVisible(assert, options={}) {
...
+  const $target = getTooltipTargetFromBody(options.targetSelector);
+  const targetPosition = $target[0].getBoundingClientRect();
+  const $tooltip = getTooltipFromBody(options.selector);
+  const tooltipPosition = $tooltip[0].getBoundingClientRect();
```
Comment (a15n) [inline test-support/helpers/ember-tooltips.js]: why use `getBoundingClientRect` instead of `$this.find('.ember-tooltip').position();`?

Comment (sir-dunxalot) [inline test-support/helpers/ember-tooltips.js]: I plan to use other values in position (e.g. width) to test `keepInWindow` and `spacing` in the future. This seemed like a less jquery-dependent way of doing things too.

File: test-support/helpers/ember-tooltips.js (hunk 1 above)
Comment (sir-dunxalot) [inline test-support/helpers/ember-tooltips.js]: Good catch. I'm going to address this in my next PR (I know it's not best practice to fix in a future PR, but I will be refactoring a little bit of this code with the `spacing` test helper).

**Pull request merged**